.github/workflows/dependency-review.yml

```diff
@@ -30,9 +30,13 @@ jobs:
           check_github_status: "true"
           # This functionality is poorly implemented and has been
           # causing problems due to the MITM implementation hogging or
-          # leaking memory.  If this happens to you just uncomment
-          # this line.
-          # monitor_permissions: "false"
+          # leaking memory.  As a result we disable it by default.  If
+          # you want to temporarily enable it, simply set
+          # monitor_permissions equal to "true".
+          #
+          # TODO: Re-anable this functionality when practical.  See
```

Comment:
Typo:
```suggestion
          # TODO: Re-enable this functionality when practical.  See
```

Reply:
I created cisagov/skeleton-generic#209 to fix this upstream.